HIP/hip-0000-hts-precompile.md

```diff
@@ -0,0 +1,299 @@
+- hip: <HIP number (this is determined by the HIP editor)>
+- title: Hyperledger Token Service Precompiled Contract for Hyperledger Smart
+  Contract Service
+- author: Danno Ferrin <danno.ferrin@gmail.com>
+- type: Standards Track
+- category: Core
+- status: Draft
+- created: 2021-11-04
+- discussions-to: &lt;a URL pointing to the official discussion thread>
```

Comment:
Please add a link to the discussion thread

Reply:
done